src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -134,6 +136,10 @@ Apply to thead to make headers not bold.
     background-color: #c4e3f3;
 }
 
+.float-left {
+    float: left;
+}
+
```

Comment:
Unless absolutely necessary, I think we shouldn't create classes that are actually just the rules. It defeats the purpose of having a class, and still violates the separation of styling concerns from HTML.

Perhaps we can create a class on the surrounding div (e.g. `".question-recommended-length"`) and apply styles using the class and child selectors (since the related DOM structure here is rather simple).